docs/OVERVIEW.md

```diff
@@ -0,0 +1,40 @@
+# Modernization Overview
+
+# Modernization Overview
```

Comment:
There's a duplication here.

This doc is less of an overview of fulcrum core and more of an overview of this approach. We should put these docs somewhere else instead of just in the docs directory directly